update.sh

```diff
@@ -3,6 +3,16 @@
 KERNEL_VERSION_RPI1=3.18.0-trunk-rpi
 KERNEL_VERSION_RPI2=3.18.0-trunk-rpi2
 
+RASPBIAN_ARCHIVE_KEY_DIRECTORY="https://archive.raspbian.org"
+RASPBIAN_ARCHIVE_KEY_FILE_NAME="raspbian.public.key"
+RASPBIAN_ARCHIVE_KEY_URL="${RASPBIAN_ARCHIVE_KEY_DIRECTORY}/${RASPBIAN_ARCHIVE_KEY_FILE_NAME}"
```

Comment:
that is totally not an answer :-)

What's wrong with just this?
`RASPBIAN_ARCHIVE_KEY_URL="https://archive.raspbian.org/raspbian.public.key"`
`RASPBIAN_ARCHIVE_KEY_FINGERPRINT="A0DA38D0D76E8B5D638872819165938D90FDDD2E"`